src/components/CurrencySymbolButton.tsx

```diff
@@ -15,16 +19,25 @@ type CurrencySymbolButtonProps = {
 };
 
 function CurrencySymbolButton({onCurrencyButtonPress, currencySymbol}: CurrencySymbolButtonProps) {
+    const { translate } = useLocalize();
     const styles = useThemeStyles();
-    const {translate} = useLocalize();
+    const theme = useTheme();
+
     return (
         <Tooltip text={translate('common.selectCurrency')}>
             <PressableWithoutFeedback
                 onPress={onCurrencyButtonPress}
                 accessibilityLabel={translate('common.selectCurrency')}
                 role={CONST.ROLE.BUTTON}
             >
+                <View style={[styles.flexRow, styles.alignItemsCenter]}>
```

Comment:
```suggestion
                <View style={[styles.flexRow, styles.alignItemsCenter, styles.gap1]}>
```
This will give us the correct 4px spacing.